docs/source/llms/tracing/index.rst

```diff
@@ -0,0 +1,558 @@
+    :alt: LangChain Tracing via autolog
+    :width: 100%
+    :align: center
+
+.. note::
+    The example above is purposely simple (a simple chat completions demonstration) for purposes of brevity. In real-world scenarios involving complex 
+    RAG chains, the trace that is recorded by MLflow will be significantly more complex and verbose. 
+
+
+Tracing Fluent APIs
+-------------------
+
+MLflow's :py:func:`fluent APIs <mlflow.start_span>` provide a straightforward way to add tracing to your functions and code blocks. By using decorators, function wrappers, 
+and context managers, you can easily capture detailed trace data with minimal code changes. This section will cover how to initiate traces 
+using these fluent APIs.
+
+Initiating a Trace
+^^^^^^^^^^^^^^^^^^
+
+In this section, we will explore different methods to initiate a trace using MLflow's fluent APIs. These methods allow you to add tracing 
+functionality to your code with minimal modifications, enabling you to capture detailed information about the execution of your functions and workflows.
+
+Trace Decorator
+###############
+
+The trace decorator allows you to automatically capture the inputs and outputs of a function by simply adding the :py:func:`@mlflow.trace <mlflow.trace>` decorator 
+to its definition. This approach is ideal for quickly adding tracing to individual functions without significant changes to your existing code.
+
+.. code-block:: python
+
+    import mlflow
+
+    # Create a new experiment to log the trace to
+    mlflow.set_experiment("Tracing Demo")
+
+
+    # Mark any function with the trace decorator to automatically capture input(s) and output(s)
+    @mlflow.trace
+    def some_function(x, y, z=2):
+        return x + (y - z)
+
+
+    # Invoking the function will generate a trace that is logged to the active experiment
+    some_function(2, 4)
+
+You can add additional metadata to the tracing decorator as follows:
+
+.. code-block:: python
+
+    @mlflow.trace(name="My Span", span_type="func", attributes={"a": 1, "b": 2})
+    def my_func(x, y):
+        return x + y
+
+When adding additional metadata to the trace decorator constructor, these additional components will be logged along with the span entry within 
+the trace that is stored within the active MLflow experiment.
+
+**What is captured?**
+
+If we navigate to the MLflow UI, we can see that the trace decorator captures the following information:
+
+- **Inputs**: In the case of our decorated function, this includes the state of all input arguments (including the default `z` value that is applied).
+- **Response**: The output of the function is also captured, in this case the result of the addition and subtraction operations.
+- **Trace Name**: The name of the decorated function.
+
+.. figure:: ../../_static/images/llms/tracing/trace-demo-1.png
+    :alt: Trace UI - simple use case
+    :width: 100%
+    :align: center
+
+Context Handler
```

Comment:
Hmm from the first read, this section might be a bit misleading🤔 We haven't mention about the child-parent relationship or exception handling until this section, so it sounds like that those capabilities are the differentiators of context manager over the decorator.

However, the only functional advantage of the context manager is that you can wrap arbitral lines of code not a function.  How about moving the examples about parent-child relationship and exceptions from the context manager section to the decorator section. Imo users should use the function decorator as much as possible as it's less error prone, and the context manager section can focus on the particular use case where function decorator cannot serve for. WDYT??

Reply:
Good idea; changed and moved